src/frontend/komponenter/Felleskomponenter/Hendelsesoversikt/komponenter/Brev.tsx

```diff
@@ -0,0 +1,114 @@
+    const [hentetForhåndsvisning, settHentetForhåndsvisning] = React.useState<Ressurs<string>>(
+        byggTomRessurs()
+    );
+    const [visModal, settVisModal] = React.useState(false);
+    const behandlingId =
+        åpenBehandling.status === RessursStatus.SUKSESS && åpenBehandling.data.behandlingId;
+    const behandlingSteg =
+        åpenBehandling.status === RessursStatus.SUKSESS && åpenBehandling.data.steg;
+
+    const sendBrev = (brevData: IBrevData) => {
+        settInnsendtBrev(byggHenterRessurs());
+        axiosRequest<string, IBrevData>({
+            method: 'POST',
+            data: brevData,
+            url: `/familie-ba-sak/api/dokument/send-brev/innhente-opplysninger/${behandlingId}`,
+        })
+            .then((response: Ressurs<string>) => {
+                settInnsendtBrev(response);
+                if (response.status === RessursStatus.SUKSESS) {
+                    settVisModal(true);
+                }
+            })
+            .catch((_error: AxiosError) => {
+                settInnsendtBrev(byggFeiletRessurs('Ukjent feil ved sending av brev.'));
+            });
+    };
+
+    const hentForhåndsvisning = (brevData: IBrevData) => {
+        settHentetForhåndsvisning(byggHenterRessurs());
+        axiosRequest<string, IBrevData>({
+            method: 'POST',
+            data: brevData,
+            url: `/familie-ba-sak/api/dokument/forhaandsvis-brev/innhente-opplysninger/${behandlingId}`,
+        })
+            .then((response: Ressurs<string>) => {
+                if (response.status === RessursStatus.SUKSESS) {
+                    settHentetForhåndsvisning(
+                        byggDataRessurs(`data:application/pdf;base64,${response.data}`)
+                    );
+                } else if (response.status === RessursStatus.FEILET) {
+                    settHentetForhåndsvisning(response);
+                } else {
+                    settHentetForhåndsvisning(
+                        byggFeiletRessurs('Ukjent feil, kunne ikke generere forhåndsvisning.')
+                    );
+                }
+            })
+            .catch((_error: AxiosError) => {
+                settHentetForhåndsvisning(
+                    byggFeiletRessurs('Ukjent feil ved henting av forhåndsvisning.')
+                );
+            });
+    };
+
+    return (
+        <div className={'brev'}>
+            {behandlingSteg && hentStegNummer(behandlingSteg) >= 2 ? (
+                <Brevskjema
+                    sendBrev={sendBrev}
+                    innsendtBrev={innsendtBrev}
+                    hentForhåndsvisning={hentForhåndsvisning}
+                    hentetForhåndsvisning={hentetForhåndsvisning}
+                />
+            ) : (
+                'Søknad må være registrert for å sende ut brev manuelt' // TODO: Hvordan skal vi håndtere dette? Man skal jo også kunne sende ut ved revurdering og ikke søknad. Skal det tas i denne omgangen?
+            )}
```

Comment:
Jeg tenker at denne modulen må være synlig hele tiden. Hvilke brev man kan sende ut bør kanskje være avhengig av behandlingen.

Reply:
Enn så lenge støtter vi kun utsending av brev innhente opplysninger ved innsendt søknad. Tenker derfor dette kan fikses når det er støtte for mer? Vi kan alternativt vise den hele tida, har bare opplevd et par ganger at vi glemmer å registrere søknaden og sitter med noe vi tror er en rar bug før vi kommer på at det ikke er implementert støtte for enda

Reply:
sendte melding på slack :) bør finnes en bedre løsning